src/ResourceManager/DeploymentManager/Commands.DeploymentManager/AzureRM.DeploymentManager.NetCore.psd1

```diff
@@ -0,0 +1,150 @@
+#
+# Module manifest for module 'AzureRM.DeploymentManager'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 8/25/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '1.0.0'
+
+# Supported PSEditions
+CompatiblePSEditions = 'Core'
+
+# ID used to uniquely identify this module
+GUID = 'caac1590-e859-444f-a9e0-62091c0f5929'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'PowerShell .Net Core] Microsoft Azure PowerShell - Deployment Manager cmdlets for Azure Resource Manager.'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '5.1'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# DotNetFrameworkVersion = ''
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = '4.0'
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile.Netcore'; ModuleVersion = '0.12.0'; })
+
+# Assemblies that must be loaded prior to importing this module
+RequiredAssemblies = '.\Microsoft.Azure.Management.DeploymentManager.dll'
+
+# Script files (.ps1) that are run in the caller's environment prior to importing this module.
+# ScriptsToProcess = @()
+
+# Type files (.ps1xml) to be loaded when importing this module
+# TypesToProcess = @()
+
+# Format files (.ps1xml) to be loaded when importing this module
+# FormatsToProcess = '.\Microsoft.Azure.Commands.DeploymentManager.format.ps1xml'
+
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+NestedModules = @('.\Microsoft.Azure.Commands.DeploymentManager.dll')
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = 'Get-AzureRmDeploymentManagerArtifactSource', 
```

Comment:
The [design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/133) does not mention any of the cmdlets here... can you kindly add them to it? Also I don't see `Get-AzureRmRollOut` here... is that going to be added later?

It will be good to see all the cmdlets in the design review so that we can get a complete picture. I will review them here as well.

Reply:
Created a new cmdlet design review to cover all the cmdlets here: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/142